### ISSUE TITLE
Update ng-select to support Screen Readers

### ISSUE THREAD

**Issue by saad-mo** [SAP/spartacus#10887]
- This ticket contains improvements for [Headings and Labels Success Criterion](https://github.com/SAP/spartacus/issues/11192)  ([WCAG 2.1 2.4.6](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html))
- master ticket: #11192

`ng-select` is a generic component shared across the application and has limited a11y support. There are workaround to make it more a11y friendly, but it may not be worth it and cleaner solutions could be to switch all `ng-select` to `native select`. 

**Screen Reader issue:** ng-select is announced as a list box, but the default value or place holder is not announced, in this case, "Relevance". ng-select should support `aria-label attribute`

<img width="1306" alt="2021-02-19_11-22-16" src="https://user-images.githubusercontent.com/15215400/108559577-19adb080-72c1-11eb-8bdb-cb52c6aafc56.png">

## Improvements 
- [x] Investigate workaround to add support for `aria-label`.

**Solution:** Upgrade ng-select to v5.10.0 which offers `aria-lable` support

**Comment by Pucek9** [comment]
Version of ng-select we uses is 5.0.9, since 5.0.10 there is added fix for aria stuff 
https://github.com/ng-select/ng-select/issues/1617
https://github.com/ng-select/ng-select/releases/tag/v5.0.10
Maybe update the package will be helpful? v5.1.0 seems as safe change

**Comment by saad-mo** [comment]
QA steps for Screen Reader Improvements: 
- Open the following pages and verify if a label renders next to the select dropdown  <img width="284" alt="Screen Shot 2021-07-28 at 11 55 41 AM" src="https://user-images.githubusercontent.com/15215400/127364521-098cb522-ae87-42d8-bac2-e06f70e94b86.png">
- Product list 
- Order history 
- Order Approval List 
- My coupon 
- Replenishment Order history
- Order return

**Comment by KateChuen** [comment]
Tested locally using develop branch, 
- On Product Listing page
supports aria-label, VO voices the dropdown menu, no AMP violation
<img width="1369" alt="Screen Shot 2022-05-18 at 1 19 49 PM" src="https://user-images.githubusercontent.com/45409515/169105101-bf975c18-aa22-4ddc-a52a-98374da5f703.png">
<img width="1066" alt="Screen Shot 2022-05-18 at 1 23 19 PM" src="https://user-images.githubusercontent.com/45409515/169105167-6d865351-4b70-4933-ad9c-9321607ece06.png">
<img width="875" alt="Screen Shot 2022-05-18 at 1 21 26 PM" src="https://user-images.githubusercontent.com/45409515/169105191-29f2c412-a3de-456c-9e7d-2106c06456a8.png">

- Order History - Orders Tab OK
<img width="1389" alt="Screen Shot 2022-05-18 at 2 26 36 PM" src="https://user-images.githubusercontent.com/45409515/169121500-66ec0124-1854-42cd-a309-f74bec7783a0.png">
<img width="845" alt="Screen Shot 2022-05-18 at 2 41 54 PM" src="https://user-images.githubusercontent.com/45409515/169121649-47123878-c5c1-4d29-a4e7-7174bc80f0dc.png">
<img width="872" alt="Screen Shot 2022-05-18 at 2 27 08 PM" src="https://user-images.githubusercontent.com/45409515/169121707-fcf5c1f7-c538-4095-84b9-d1269cb0ef7e.png">

- Order History - Returns Tab -> 1 AMP Violation, opened a ticket: https://jira.tools.sap/browse/CXSPA-450
<img width="1361" alt="Screen Shot 2022-05-18 at 2 33 44 PM" src="https://user-images.githubusercontent.com/45409515/169121874-6e67b0f9-4d95-4410-99be-6e20e468bde4.png">
<img width="1202" alt="Screen Shot 2022-05-18 at 2 43 56 PM" src="https://user-images.githubusercontent.com/45409515/169122234-a50a8347-c213-4213-88b2-ee9d4981ab69.png">
<img width="1250" alt="Screen Shot 2022-05-18 at 2 34 41 PM" src="https://user-images.githubusercontent.com/45409515/169122337-ecbb8da8-6a61-4db8-8492-59a262e5f90c.png">

- My Coupons
<img width="1438" alt="Screen Shot 2022-05-18 at 2 54 41 PM" src="https://user-images.githubusercontent.com/45409515/169133330-0cec999a-caa7-47e6-8e21-1a73acd2caa5.png">
<img width="865" alt="Screen Shot 2022-05-18 at 2 56 57 PM" src="https://user-images.githubusercontent.com/45409515/169133549-bd6b4666-c6b3-48b3-a828-81d69b606bcf.png">
<img width="1278" alt="Screen Shot 2022-05-18 at 2 55 19 PM" src="https://user-images.githubusercontent.com/45409515/169133648-db12611f-5a80-4f4b-aeba-78865b6361b1.png">

- Replenishment Order, 1 AMP violation, opened ticket https://jira.tools.sap/browse/CXSPA-475
<img width="1395" alt="Screen Shot 2022-05-18 at 3 05 35 PM" src="https://user-images.githubusercontent.com/45409515/169138478-2811c1bb-b99e-4a3f-a1b6-6fbce1832a5a.png">

<img width="1125" alt="Screen Shot 2022-05-18 at 3 08 26 PM" src="https://user-images.githubusercontent.com/45409515/169138395-69aad234-6b97-4e42-80f2-43d6be3a850e.png">

<img width="1254" alt="Screen Shot 2022-05-18 at 3 07 00 PM" src="https://user-images.githubusercontent.com/45409515/169140488-ce9330d9-2988-4af3-a95a-5b6d52a905d6.png">


- Order Approval List, 3 AMP violations, opened ticket https://jira.tools.sap/browse/CXSPA-476
<img width="1430" alt="Screen Shot 2022-05-18 at 3 11 07 PM" src="https://user-images.githubusercontent.com/45409515/169138359-f08248e2-aac0-41c2-8a63-3b8a6194543b.png">

<img width="798" alt="Screen Shot 2022-05-18 at 3 12 57 PM" src="https://user-images.githubusercontent.com/45409515/169138320-6275a8ee-f2e5-42cc-855f-86ce55f667b1.png">

<img width="1258" alt="Screen Shot 2022-05-18 at 3 11 59 PM" src="https://user-images.githubusercontent.com/45409515/169140828-db5d8f67-0f2c-439d-8fb2-8f69c492fb3f.png">
